### PR TITLE
Fix: Make FilePicker work with Firefox

### PR DESCRIPTION
**Pull request by vinteo**

### CODE REVIEW

File: test/components/adslotUi/FilePickerComponentTest.js [hunk 1]
```diff
@@ -20,10 +24,14 @@ describe('FilePickerComponent', () => {
...
     const onSelect = () => { onSelectCalls++; };
 
-    const component = shallow(<FilePickerComponent onSelect={onSelect} />);
+    // mount is needed for refs
+    const component = mount(<FilePickerComponent onSelect={onSelect} />);
     let fileElement = component.find('.form-control');
     expect(fileElement.prop('title')).to.equal('');
 
+    const selectButtonElement = component.find(Button);
+    selectButtonElement.simulate('click');
```
Comment (OndrejCholeva) [inline test/components/adslotUi/FilePickerComponentTest.js]: can we expect something to happen on click?

**Comment by OndrejCholeva** [comment]
LGTM, works fine in FF, Chrome, Safari, IE

**Comment by pphminions** [comment]
LGTM 🎸

**Comment by davepoon** [comment]
LGTM!!

**Comment by vinteo** [comment]
what happend to code cov? from the build logs:
```
=============================== Coverage summary ===============================
Statements   : 100% ( 1024/1024 ), 34 ignored
Branches     : 100% ( 358/358 ), 79 ignored
Functions    : 100% ( 193/193 ), 2 ignored
Lines        : 100% ( 471/471 )
================================================================================
```

**Comment by vinteo** [comment]
@OndrejCholeva @pphminions @davepoon merge?

**Pull request merged**